tools/cli/adminConfigStoreCommands.go

```diff
@@ -186,9 +185,7 @@ func AdminListDynamicConfig(c *cli.Context) {
 	ctx, cancel := newContext(c)
 	defer cancel()
 
-	req := &types.ListDynamicConfigRequest{
-		ConfigName: dynamicconfig.UnknownKey.String(),
-	}
+	req := &types.ListDynamicConfigRequest{}
```

Comment:
I'm a little inclined to say "use `ConfigName: nil`"... but also kinda meh since there's only the one field.
an explicit nil in a field "feels" a bit more intentional to me tho, so I lean that way since the nil is intended, not just "an empty request".